### PR TITLE
add new api install_or_update for web extension

### PR DESCRIPTION
**Pull request by DiceyClickcorp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,41 +1 @@
...
-```
-
-Not using SSH:
-
-```
-$ GIT_USER=<Your GitHub username> yarn deploy
-```
-
-If you are using GitHub pages for hosting, this command is a convenient way to build the website and push to the `gh-pages` branch.
+[ClickCorp](https://www.clickcorp.com) builds a new generation automation framework **Clicknium** to support GUI automation for all types of applications. Automation development efficiency is highly improved with features of Clicknium, including developer tooling support and cloud native data management for colaboration.
```
Comment (robinbob) [inline README.md:1]: collaboration

File: doc/api/python/webdriver/webextension/install_or_update.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+sidebar_label: install_or_update
+---
+# WebDriver.extension.install_or_update
+
+***def install_or_update(self) -> bool*** 
+
+Install or update browser extension based on current detected extension status.
+
+**Returns:** bool  
+    &emsp;Return `True` means the extension has been installed or updated, `False` means the extension has already installed the lastest version.
```
Comment (robinbob) [inline doc/api/python/webdriver/webextension/install_or_update.md:12]: `True` means the extension is being installed or updated, `False` means the latest version extension has already been installed before calling this function.

File: README.md (hunk 1 above)
Comment (DiceyClickcorp) [reply README.md]: done

File: doc/api/python/webdriver/webextension/install_or_update.md (hunk 2 above)
Comment (DiceyClickcorp) [reply doc/api/python/webdriver/webextension/install_or_update.md]: done

**Review by robinbob — APPROVED** [review]

**Pull request merged**